components/service/glean-gradle-plugin/build.gradle

```diff
@@ -0,0 +1,44 @@
+plugins {
+    id 'groovy'
+    id 'maven-publish'
+}
+
+dependencies {
+    implementation gradleApi()
+    implementation localGroovy()
+}
+
+publishing {
+    publications {
+        maven(MavenPublication) {
+            from components.java
+
+            pom {
+                groupId = config.componentsGroupId
+                artifactId = archivesBaseName
+                description = project.ext.description
+                version = config.componentsVersion + (project.hasProperty('snapshot') ? '-SNAPSHOT' : '')
+
+                licenses {
+                    license {
+                        name = libLicense
+                        url = libLicenseUrl
+                    }
+                }
+
+                developers {
+                    developer {
+                        name = 'Mozilla Android Components Team'
+                        email = 'android-components@lists.mozilla.org'
```

Comment:
I think we should either add more than 1 `developer` section or just put the Glean team here.

Reply:
I see, good point. Let's not touch it then.

Reply:
Looking into this closer, I think it's fine to update this to say `glean-team@mozilla.com`.